public/app/plugins/datasource/prometheus/components/monaco-query-field/monaco-completion-provider/index.ts

```diff
@@ -74,7 +74,7 @@ export function getCompletionProvider(
   };
 
   return {
-    triggerCharacters: ['{', ',', '[', '(', '=', '~', ' '],
+    triggerCharacters: ['{', ',', '[', '(', '=', '~', ' ', '"'],
```

Comment:
Prometheus supports string wrapped `'` as well (and maybe backticks?). Could it be added?

Reply:
i added it to the todolist-monaco-issue. it can be added but i also need to fix an another place, where we parse the promql query, there i don't parse singlequoted (or backtick-quoted) strings correctly currently.